client/src/pages/MoviesList/utility.js

```diff
@@ -0,0 +1,19 @@
+import { useState } from 'react'
+import { useConfiguration } from '../../providers/Configuration'
+import { authFetch } from '../../auth'
+
+export const useTurns = () => {
+  const { apiUrl } = useConfiguration()
+  const [houseTurns, setHouseTurns] = useState()
+
+  async function getHouseTurns(houseId) {
```

Comment:
See my comment above about relying on the presence of `houseTurns` to derive loading state.
You probably want to set a loading and error state in this hook.